lua/testaustime.lua

```diff
@@ -35,6 +35,10 @@ function sendheartbeat()
 
     local hb = getheartbeatdata()
 
+    if vim.fn.buflisted(vim.fn.bufnr("%")) == 0 then
+        return
+    end
```

Comment:
This should probably use `0` which is a magic id for the current buffer instead of `vim.fn.bufnr("%")`

Reply:
Doesn't seem to give the same results (atleast on my installation):
1. Open a single buffer
2. Run `:lua vim.fn.buflisted(vim.fn.bufnr("%"))` vs run `:lua vim.fn.buflisted(0)`
3. Open another buffer and both give the same result.

Reply:
aamunen case, merging